milli/src/update/settings.rs

```diff
@@ -580,6 +580,23 @@ impl<'a, 't, 'u, 'i> Settings<'a, 't, 'u, 'i> {
     fn update_exact_words(&mut self) -> Result<()> {
         match self.exact_words {
             Setting::Set(ref mut words) => {
+                fn normalize(analyzer: &Analyzer<&[u8]>, text: &str) -> String {
+                    analyzer.analyze(text).tokens().map(|token| token.text().to_string()).collect()
```

Comment:
Ok, so as I understand this function you are using the tokenizer to split and normalize the word for you, concatenating all of the normalized words together if there is more than one word. Is it expected?

Reply:
yes, for now @gmourier think it's better to ignore this edge case